db_load_data/load_customer_data.py

```diff
@@ -17,8 +17,6 @@ def load_customer_data(conn, csv_file_path):
         print("Customer data loaded successfully.")
     except Exception as e:
         print(f"Error loading data: {e}")
-    finally:
-        conn.close()
```

Comment:
Close the connection, recommended.

Reply:
closed in load_data.py, using the same conn to load all data